### ISSUE TITLE
feature request: save login password

### ISSUE THREAD

**Issue by EleanorFox** [ttalvitie/browservice#129]
So I started using browservice today and I noticed that if I close the borwser or tab I will have to log in to every site that I logged it again which is kinda frustrated so if you could add this feature that would be great. Thankyo so much and have a nice day :D

**Comment by robinnik1** [comment]
I have the same issue I tried the "--data-dir=PATH" command and don't work can you please help us

**Comment by ttalvitie** [comment]
Indeed, using `--data-dir` should be the correct solution; you should set it to use the same data directory every time you run Browservice, and this data directory will store your cookies, meaning that you stay logged in to the websites.

The help for `--data-dir` as shown by `--help`:
```
  --data-dir=PATH                       absolute path to a directory that will be used to store data
                                          such as cookies and cache; if empty, the browser runs in
                                          incognito mode [default empty]
```

For example, on my Linux machine I would point it to a path below my home directory every time I run Browservice:
```
./browservice-v0.9.11.0-x86_64.AppImage --data-dir=/home/topi/browservicedata
```
On the first run, it will create the subdirectory `/home/topi/browservicedata` to my home directory `/home/topi`.

Note that you the path you give to `--data-dir` must be absolute. If I give it a non-absolute path (doesn't start with root `/` or on Windows the drive letter such as `C:\`), it will not work. The embedded Chromium will show this with an error message (but still keep running as if `--data-dir` was not given):
```
[0114/230810.010343:ERROR:context.cc(106)] The cache_path directory (browservicedata) is not an absolute path. Defaulting to empty.
[0114/230810.015784:ERROR:context.cc(179)] The cache_path is invalid. Defaulting to in-memory storage.
```

It will also fail if it cannot create the directory, for example if I try to create it somewhere where I don't have write permissions:
```
[0114/231316.361801:ERROR:context.cc(164)] The cache_path directory (/asd) could not be created.
[0114/231316.368988:ERROR:context.cc(179)] The cache_path is invalid. Defaulting to in-memory storage.
```

Make sure that you don't see error messages like the messages above in the Browservice console output; otherwise, something is wrong with the path you gave to `--data-dir`.
